--- sdk/sdk/src/rplidar_driver.cpp.orig	2019-10-15 05:31:18.000000000 +0200
+++ sdk/sdk/src/rplidar_driver.cpp	2025-05-12 09:11:54.686094393 +0200
@@ -1790,7 +1790,7 @@
 {
     DEPRECATED_WARN("grabScanData()", "grabScanDataHq()");
 
-    switch (_dataEvt.wait(timeout))
+    switch (static_cast<int>(_dataEvt.wait(timeout)))
     {
     case rp::hal::Event::EVENT_TIMEOUT:
         count = 0;
@@ -1819,7 +1819,7 @@
 
 u_result RPlidarDriverImplCommon::grabScanDataHq(rplidar_response_measurement_node_hq_t * nodebuffer, size_t & count, _u32 timeout)
 {
-    switch (_dataEvt.wait(timeout))
+    switch (static_cast<int>(_dataEvt.wait(timeout)))
     {
     case rp::hal::Event::EVENT_TIMEOUT:
         count = 0;